ai_ta_backend/vector_database.py

```diff
@@ -1138,18 +1252,19 @@ def get_stuffed_prompt(self, search_query: str, course_name: str, token_limit: i
       token_counter, _ = count_tokens_and_cost(pre_prompt + '\n\nNow please respond to my query: ' + search_query) # type: ignore
       valid_docs = []
       for d in found_docs:
-        if "pagenumber" not in d.payload.keys(): # type: ignore
-          d.payload["pagenumber"] = d.payload["pagenumber_or_timestamp"] # type: ignore
-        doc_string = f"---\nDocument: {d.payload['readable_filename']}{', page: ' + str(d.payload['pagenumber']) if d.payload['pagenumber'] else ''}\n{d.payload.get('page_content')}\n" # type: ignore
+        if "pagenumber" not in d.payload["metadata"].keys(): # type: ignore

```

Comment:
Similarly, all this should stay with no `["metadata"]` that is deprecated.

Reply:
I'll check this. Need to do a `git merge main` to sync everything.